packages/dashboard-backend/src/devworkspace-client/services/api/serverConfigApi.ts

```diff
@@ -95,4 +95,18 @@ export class ServerConfigApi implements IServerConfigApi {
       );
     }
   }
+
+  async getRunningWorkspacesLimit(): Promise<number> {
+    try {
+      const cheCustomResource = await this.getCheCustomResource();
+
+      return cheCustomResource.spec.server.runningWorkspacesLimit || 1;
```

Comment:
@olexii4 also, since there is a discussion about the che-server deprecation, I would propose to change the structure to `spec.devworkspace.runningLimit`

Reply:
Yes, the default value is 1.

Ok. I will change 'runningWorkspacesLimit' -> 'runningLimit'.

Reply:
@olexii4 thanks, I have provided a suggestion below